### PR TITLE
Country api tests

### PR DESCRIPTION
**Pull request by acrobat**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

**Comment by tvlooy** [comment]
please re-test, the PR is ok

**Comment by acrobat** [comment]
Force pushed to trigger the builds again!

**Comment by pjedrzejewski** [comment]
Awesome work guys, thank you so much Jeroen! :+1:

File: src/Sylius/Bundle/CoreBundle/Tests/Controller/CountryApiTest.php [hunk 1]
```diff
@@ -0,0 +1,131 @@
+<?php
+
```
Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Tests/Controller/CountryApiTest.php]: Missing license block

Comment (acrobat) [inline src/Sylius/Bundle/CoreBundle/Tests/Controller/CountryApiTest.php]: Fixed in #4475

File: src/Sylius/Bundle/CoreBundle/Tests/Controller/CountryApiTest.php [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        $response = $this->client->getResponse();
+        $this->assertResponse($response, 'country/create_validation_fail_response', Response::HTTP_BAD_REQUEST);
+    }
+
+    public function testCreateCountryResponse()
+    {
+        $this->loadFixturesFromFile('authentication/api_administrator.yml');
+
+        $data =
+            <<<EOT
```
Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Tests/Controller/CountryApiTest.php]: I would suggest to put both `eot` keywords in the same line

File: src/Sylius/Bundle/CoreBundle/Tests/Controller/CountryApiTest.php (hunk 1 above)
Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Tests/Controller/CountryApiTest.php]: :+1:

**Pull request merged**